functions/helpers.bash

```diff
@@ -356,7 +356,9 @@ tryUntil() {
 
   until [[ $attempts -le 0 ]]; do
     cond_echo "\\nexecuting $cmd \\c"
-    if [[ $(eval "$cmd") -eq 0 ]]; then break; fi
+    eval "$cmd"
+    out=$?
```

Comment:
Seems to work, what caused the change to be needed?

Reply:
it didn't properly evaluate the test I use for inet availability

Reply:
Fair enough.